src/hotspot/share/code/aotCodeCache.hpp

```diff
@@ -331,6 +370,9 @@ class AOTCodeCache : public CHeapObj<mtCode> {
   static bool is_dumping_adapters() NOT_CDS_RETURN_(false);
   static bool is_using_adapters() NOT_CDS_RETURN_(false);
 
+  static bool is_dumping_stubs() NOT_CDS_RETURN_(false);
+  static bool is_using_stubs() NOT_CDS_RETURN_(false);
```

Comment:
We have singular naming (`is_dumping_stub()`) for these methods in `premain` branch.
I was debating to do separate RFE for renaming in mainline or may be you can do it here.
It is up to you.
I did not pay attention to these when I work on adapter caching. But now I have to merge from mainline to premain and I noticed difference.

Reply:
I will make this change.